packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -51,9 +50,14 @@ export const SimpleDocument: FC<Props> = ({
       if (highlight && isPassage(highlight)) {
         passage = highlight as QueryResultPassage;
         field = passage.field;
+        if (!field) {
+          // if passage has no defined field, choose a default and unset `highlight`
+          field = 'text';
+          highlight = undefined;
+        }
```

Comment:
Since the SDK defines `field` as an optional prop for `QueryResultPassage`, had to add this fallback code.

Reply:
how was this working before?

Reply:
Don't know. I was wondering the same thing. Might be as a result of updated dependencies, which are now working better at detecting certain issues.